etna/models/nn/deepstate/state_space_model.py

```diff
@@ -0,0 +1,445 @@
+from abc import ABC
+from abc import abstractmethod
+from typing import Callable
+from typing import List
+from typing import Optional
+from typing import Union
+
+import numpy as np
+import pandas as pd
+import torch
+from torch import Tensor
+from torch.nn.functional import one_hot
+
+from etna.core import BaseMixin
+
+
+class SSM(ABC, BaseMixin):
+    """Base class for State Space Model.
+
+    The system dynamics is described with the following equations:
```

Comment:
<img width="556" alt="image" src="https://github.com/tinkoff-ai/etna/assets/33594071/45090f1b-7256-428b-bc0a-8c32921283a7">

It seems not rendering correctly